plugins/basic/bootstrap.py

```diff
@@ -66,11 +66,10 @@ def plugin_startup():
     except:
         import traceback
         
-        # Only log in the client as we might have gotten a "connection reset" 
-        # exception
-        import sg_client
-        sg_client.log('Shotgun Toolkit Error:')
-        sg_client.log('Exception stack trace:\n\n{}'.format(traceback.format_exc()))
+        # Only log in bootstrap incase of an exception
```

Comment:
"Only log" does not apply here. This was a workaround for connection issues with rpyc.
Please remove this comment as it is confusing